### PR TITLE
doc(OFV): Document how to configure cloud-init

### PR DESCRIPTION
**Pull request by holmanb**
## Proposed commit message
```
doc(OFV): Document how to configure cloud-init
```

## Additional context

https://github.com/canonical/cloud-init/issues/5517

## Merge type

- [x] Squash merge using "Proposed Commit Message"
- [ ] Rebase and merge unique commits. Requires commit messages per-commit each referencing the pull request number (#<PR_NUM>)

### CODE REVIEW

**Review by aciba90 — APPROVED** [review]
LGTM, thanks!

File: doc/rtd/reference/datasources/ovf.rst [hunk 1]
```diff
@@ -3,9 +3,35 @@
...
 ***
 
-The OVF datasource provides a datasource for reading data from an
+The OVF datasource provides a generic datasource for reading data from an
 `Open Virtualization Format`_ ISO transport.
 
+What platforms support OVF
+--------------------------
+
+OFV is an open standard which is supported by various virtualization
```
Comment (s-makin) [inline doc/rtd/reference/datasources/ovf.rst:12]: Sorry I was a bit late to the party, but OFV s/b OVF :)

Comment (aciba90) [reply doc/rtd/reference/datasources/ovf.rst:12]: Ups, I missed that one. The commit message also contained OFV.

**Comment by TobiPeterG** [comment]
Hey thanks. :)
This makes it clearer, however, by default the cloud image I use doesn't have the vapp property network-config.
Could this be added to the ovf image?

**Comment by aciba90** [comment]
> Could this be added to the ovf image?

Hi, @TobiPeterG. The OVF image is not provided by cloud-init nor Canonical, I believe, but by the cloud you are running on.

**Comment by aciba90** [comment]
@TobiPeterG

ova: Ubuntu image
ovf: dynamic disc with user-data / vendor-data / network config

The network config, as it is not static, has to be provided via the OVF disc, and that is build and presented by the cloud to the instance.

**Comment by holmanb** [comment]
> Could this be added to the ovf image?

I looked internally at getting a field added to the default OVF provided by the official ubuntu OVA downloads. It sounds like such a change comes with a high risk of breaking existing users of this OVA, so there is little interest in modifying the default.

@TobiPeterG You are free to copy the OVF file and add the field (copy the user-data field and rename it) if you are launching directly from an OVA, or if it is cloud-provided @aciba90 suggested, your cloud will need to provide the modification.

**Pull request merged**